contracts/mixnet/src/transactions.rs

```diff
@@ -99,6 +99,7 @@ pub(crate) fn try_add_mixnode(
         layer,
         env.block.height,
         mix_node,
+        None,
```

Comment:
Why is profit margin always set to `None`? Shouldn't we require people to set their profit margins and just have the UI choose a default for them? (Something like the `advanced` section we currently have in the web wallet)

Reply:
Its `None` for backward compatibility, so that this can run against the QA net and Milhon, without issue. Once we get to updating the mixnode bonding cli we can get rid of the Option

Reply:
Created an issue